test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestJcmd.java

```diff
@@ -163,6 +173,48 @@ public void testClassHistogramAll() throws IOException {
 		log(EXPECTED_STRING_FOUND);
 	}
 
+	@Test
+	public void testDumps() throws IOException {
+		@SuppressWarnings("nls")
+		String[][] commandsAndDumpTypes = {{DUMP_HEAP, "Heap"}, {GC_HEAP_DUMP, "Heap"}, {DUMP_JAVA, "Java"}, {DUMP_SNAP, "Snap"}, {DUMP_SYSTEM, "System"}};
+		for (String[] commandAndDumpName: commandsAndDumpTypes) {
+			TargetManager tgt = new TargetManager(TestConstants.TARGET_VM_CLASS, null,
+					Collections.emptyList(), Collections.singletonList("-Xmx10M")); //$NON-NLS-1$
+			tgt.syncWithTarget();
+			String targetId = tgt.targetId;
+			assertNotNull(targetId, "target did not launch"); //$NON-NLS-1$
+			String dumpType = commandAndDumpName[1];
+			@SuppressWarnings("nls")
```

Comment:
This annotation might be better applied to the method (or the class); mixing both annotations and `//$NON-NLS-` comments seems wrong (unless they're a small minority of cases).

Reply:
I believe the preferred practice is to add the metacomments individually where practical.I used the annotation where there is a large number of literals on a line to avoid a large number of metacomments.

Reply:
I don't think we have any NLS concerns in test code: don't bother with the metacomments or add an annotation to the class.

Reply:
@DanHeidinga @pshipton Do you concur with @keithc-ca's request?